actions/server.search.all.servers.yaml

```diff
@@ -0,0 +1,65 @@
+---
+description: Query servers
+enabled: true
+entry_point: src/server_query_actions.py
+name: server.search.all.servers
+parameters:
+  timeout:
+    default: 5400
+  submodule:
+    default: search_all_servers
+    type: string
+    immutable: true
+  cloud_account:
+    description: The clouds.yaml account to use whilst performing this action
+    required: true
+    type: string
+    default: "dev"
+    enum:
+      - "dev"
+      - "prod"
+  properties_to_select:
```

Comment:
I can't see a property to manipulate e.g. if I wanted to find servers belonging to a `image_id` for example?

Reply:
this action currently lists all servers - this action just limits which property the output table shows for each server
I can alter this action to allow you to search by property - I was going to create separate actions for this though i.e. search.servers.with.property where property is something people would search for

Reply:
Hmmm, then maybe this should be called:
`list.server.all_projects` to bikeshed further?

I think we should document all the verbs, such as `list`, `search`, `notify` or `email` ....etc. in the README

Reply:
I'm going change this action to emulate the one that existed before `server.list`.
so this function will: 
 - allow running generic queries on all servers - property `equal_to`, not `equal_to` etc. 
 - allow running string queries on all servers - `any_in`, `not_any_in`